packages/core/upload/admin/src/pages/App/MediaLibrary/components/tests/__snapshots__/BulkDeleteButton.test.js.snap

```diff
@@ -97,8 +96,8 @@ exports[`BulkDeleteButton renders 1`] = `
   height: 2rem;
   padding-left: 16px;
   padding-right: 16px;
-  border: 1px solid #f5c0b8;
-  background: #fcecea;
+  border: 1px solid undefined;
+  background: undefined;
```

Comment:
🤔 

Mmmh, this file doesn't look good.

Reply:
hmmm you're right. that's suspicious. I'll look into it.

Reply:
Opened a PR – https://github.com/strapi/design-system/pull/857

Great spot  🦅 👁️